src/ClientUtilities/util/TestDomain.cs

```diff
@@ -33,6 +33,11 @@ public class TestDomain : ProxyTestRunner, TestRunner
 		/// </summary>
 		private DomainAgent agent;
 
+        /// <summary>
```

Comment:
Don't worry about the formatting in v2. We had a lot of PR's like this in the early days of v3. There we fixed it as we went, but I think we will want to keep v2 limited to any backported fixes themselves.